### ISSUE TITLE
Fully migrate to GitHub Actions

### ISSUE THREAD

**Issue by underwoo** [NOAA-GFDL/FMS#630]
**Describe the solution you'd like**
It is time to migrate away from Travis CI to GitHub actions.  The PR #339 should have the base GitHub Action tests.  Once that is merged into master, the Travis CI test can be removed.  As the Travis CI test is required, removing it may require some changes in the repository settings.

**Comment by thomas-robinson** [comment]
@underwoo I don't think I can change the settings for travis.  I looked in my settings tab, and the things I can do seem to be very limited.  I think you will have to do it.

**Comment by underwoo** [comment]
> @underwoo I don't think I can change the settings for travis.  I looked in my settings tab, and the things I can do seem to be very limited.  I think you will have to do it.  

You will soon. I'll let you know when the change has been made.

**Comment by underwoo** [comment]
@thomas-robinson you now have admin permission on the repository.

**Comment by rem1776** [comment]
Closing this because github actions was merged in with #339
